SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -406,6 +406,57 @@ FSpatialNetGUIDCache::FSpatialNetGUIDCache(USpatialNetDriver* InDriver)
 {
 }
 
+typedef TMap<UObject*, uint32> FSubobjectToOffsetMap;
+
+FORCEINLINE FSubobjectToOffsetMap CreateOffsetMapFromActor(Worker_EntityId EntityId, AActor* Actor, const FClassInfo& Info)
```

Comment:
nit - no need for inline unless performance dictates

Reply:
Can Actor be const?

Reply:
Looks like EntityId wasn't needed after all

Reply:
1. Sure, removed the specifier - the compiler can decide whether to inline this or not.
2. No, not really. `StaticFindObjectFast` requires a non-const object, and I'd rather avoid `const_cast`ing.
3. Nice spot - removed EntityId.